docs/data-collection/scanning.md

```diff
@@ -102,7 +102,7 @@
 
     ???+ quote "Participant of Cohort II"
 
-        Hey [NAME], the next block is a bit long, around 30 minutes.
+        Hey [NAME], the next block is a bit long, around 20 minutes.
```

Comment:
The DWI block is closer to 30 minutes. This will need to be considered in the new psychopy task.
```suggestion
        Hey [NAME], the next block is a bit long, around 30 minutes.
```